glean-core/src/metrics/experiment.rs

```diff
@@ -0,0 +1,113 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+use serde::{Deserialize, Serialize};
+use std::collections::HashMap;
+
+use crate::error_recording::{record_error, ErrorType};
+use crate::metrics::Metric;
+use crate::metrics::MetricType;
+use crate::storage::StorageManager;
+use crate::CommonMetricData;
+use crate::Glean;
+use crate::Lifetime;
+
+// FIXME: this should be shared?
```

Comment:
Shared with what?

Reply:
With the functions in `StorageManager` using the same constant :(